runtime/src/main/java/org/corfudb/runtime/view/TableRegistry.java

```diff
@@ -393,19 +419,18 @@ public static String getFullyQualifiedTableName(TableName tableName) {
             // To do so, consult the TableRegistry for an entry which indicates the table exists.
             if (registryTable.containsKey(
                     TableName.newBuilder()
-                    .setNamespace(namespace)
-                    .setTableName(tableName)
-                    .build())
+                            .setNamespace(namespace)
+                            .setTableName(tableName)
+                            .build())
             ) {
                 // If table does exist then the caller must use the long form of the openTable()
                 // since there are too few arguments to open a table not seen by this runtime.
                 throw new IllegalArgumentException("Please provide Key, Value & Metadata schemas to re-open"
-                + " this existing table " + tableName + " in namespace " + namespace);
+                        + " this existing table " + tableName + " in namespace " + namespace);
             } else {
-                // If the table is completely unheard of return NoSuchElementException
-                throw new NoSuchElementException(
-                        String.format("No such table found: namespace: %s, tableName: %s",
-                        namespace, tableName));
+                // If the table is completely unheard of return NoSuchElementException.
+                throw new NoSuchElementException(String.format(
+                        "No such table found: namespace: %s, tableName: %s", namespace, tableName));
```

Comment:
Not related to this PR per-se but should these types be validated when `registryTable` is populated ? So `TableRegistry` should try to load the types with the class loader and should fail with ClassNotFoundException if the type is not there, which is more appropriate than NoSuchElementException

Reply:
So the contract of CorfuStore when they designed it is that clients have to first explicitly register this table by calling openTable() and later other clients use getTable() to use the opened table.